syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -631,22 +642,28 @@ def float_prec_(self):
         if hasattr(tensor, "child"):
             self.child = tensor.child
         elif self._is_parameter():
+            self.is_wrapper = False
             self.data = tensor
+            self.data.is_wrapper = False
         else:
             del self.child
             self.set_(tensor)
+            self.is_wrapper = False
         return self
 
     float_precision_ = float_prec_
 
-    def fix_prec(self, *args, storage="auto", field_type="int100", **kwargs):
+    def fix_prec(self, *args, storage="auto", field_type="int100", no_wrap: bool = False, **kwargs):
```

Comment:
could you add a docstring to document the parameters?

Reply:
Sure!